nucliadb_models/nucliadb_models/metadata.py

```diff
@@ -292,6 +307,15 @@ def from_message(cls: Type[_T], message: resources_pb2.UserFieldMetadata) -> _T:
             including_default_value_fields=True,
             use_integers_for_enums=True,
         )
+        value["selections"] = [
+            MessageToDict(
+                selections,
+                preserving_proto_field_name=True,
+                including_default_value_fields=True,
+                use_integers_for_enums=True,
```

Comment:
don't we want to serialize the enums with the string instead of with the integer?

Reply:
Not sure about this, I'm using the same `MessageToDict` as above